client/src/main/resources/lang/messages_de_DE.properties

```diff
@@ -0,0 +1,24 @@
+chat.button.scroll=Herunterscrollen
+lobby.create.button.cancel=Abbrechen
+lobby.create.button.ok=OK
+lobby.create.label.name=Lobbyname:
+lobby.room.button.leave=Lobby verlassen
+login.button.connect=Verbinden
+login.button.login=Anmelden
+login.button.register=Registrieren
+login.label.email=Email:
+login.label.password=Passwort:
+login.label.port=Port:
+login.label.server=Server:
+main.lobby.button.create=Lobby erstellen
+main.lobby.button.join=Lobby beitreten
+main.lobby.label.name=Name:
+main.lobby.label.password=Passwort:
+main.lobby.prompt.searchField=Lobbies durchsuchen
+main.menu.button.logout=Abmelden
+main.menu.button.quit=Beenden
+main.menu.button.settings=Einstellungen
+title.lobby=Lobby
+title.login=Anmeldung
+title.main=Hauptmen�
```

Comment:
There seems to be an encoding problem here.

Reply:
Done!